pyproject.toml

```diff
@@ -1,12 +1,20 @@
 [build-system]
+build-backend = "rapids_build_backend.build"
 requires = [
-    "cmake>=3.24.0,!=3.30.0",
+    "rapids-build-backend>=0.3.2,<0.4.0.dev0",
+    "scikit-build-core[pyproject]>=0.10.0",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit dependencies.yaml and run `rapids-dependency-file-generator`.
+
+[tool.rapids-build-backend]
+build-backend = "scikit_build_core.build"
+dependencies-file = "dependencies.yaml"
+commit_files = ["legateboost/GIT_COMMIT"]
```

Comment:
`rapids-build-backend` default to adding a `GIT_COMMIT` file, which is fine.  But because the project name legate-boost and the module name `legateboost` mismatch, we have to add it explicitly (I think).